index.html

```diff
@@ -712,23 +712,23 @@ <h2 class='informative'>Vary</h2>
       <h2>Resource Authorization</h2>
       <p>
         To configure access control restrictions, implementations MUST follow the recommendations
-        of [[!SOLIDWEBAC]].
+        of [[!SOLIDWEBAC]] with the following additional requirements:
       </p>
-      <p>
-        Implementations supporting access control MUST use the [[!WEBAC]] model to
-        configure that functionality. Any access control resource in use MUST be a <a>LDP-RS</a>,
-        which SHOULD be located in the same server as the controlled resource. Implementations
-        MUST support extended control relationships in which the access control resource for some
-        resource has its own access control resource and so on. Implementations MUST make access
```

Comment:
In order to ensure the use of `Link: rel="acl"`, I would prefer to keep the follow language in the spec:
> Implementations MUST make access control resources discoverable via the <code>Link: rel="acl"</code> HTTP header, as discussed <a href="https://github.com/solid/web-access-control-spec#acl-resource-location-discovery">here</a>